data/events.txt

```diff
@@ -1126,7 +1126,55 @@ event "navy using mark ii ships"
 			"Rainmaker (Mark II)" 2
 			"Gunboat (Mark II)"
 
+	fleet "Small Oathkeeper"
+		add variant 6
```

Comment:
Is there any reason for why the frequencies of these ships differs from those above? Why not just have the same additions to the Oathkeeper fleets as the other Navy fleets get?

Reply:
I was indeed going for a lower frequency, but matching them is fine with me. Done in 8e4de9b.